scripts/test/AbinsCalculateSPowder2DTest.py

```diff
@@ -1,92 +0,0 @@
-from __future__ import (absolute_import, division, print_function)
```

Comment:
What's the reason for deleting tests?

Reply:
@martyngigg At this version of AbiINS there is no 2D map and this test is obsolete. I planning to add 2dMAp implementation with corresponding system test. So instead of keeping this unit test I plan to replace it with the system test for 2D map in the future.

Reply:
:+1: